### PR TITLE
Serves new content if config.json is changed

### PR DESCRIPTION
**Pull request by dcfix**
Fix for issue #13

### CODE REVIEW

**Comment by zpao** [comment]
This is looking good, but a couple comments first:

- We're going to do the js server in #17 
- As you can see in the discussion there, we actually also want to make sure we re-read the file when `POST`ing, otherwise we could lose data while writing to disk.

So if you could make the right changes in the other implementations, that would be awesome!

**Comment by dcfix** [comment]
I'll figure out how to reset my fork and get it submitted. Should I submit
the python and ruby files as one pull request or two?

Thanks,
Darren

**Comment by zpao** [comment]
Keeping it in this PR is fine.

Also, it looks like our bot wasn't set up right for this repo. Can you sign the CLA - https://code.facebook.com/cla? We'll need that before we can merge the changes.

File: server.rb [hunk 1]
```diff
@@ -26,6 +26,7 @@
   end
 
   # always return json
+  comments = react_version = JSON.parse(File.read('./_comments.json'))
```
Comment (zpao) [inline server.rb]: This should probably be before the handling of POST (we could also drop the global `comments` and `react_version`)

**Comment by zpao** [comment]
Ah, picked this up with a combo of other changes tonight. But if we missed anything or you have other suggestions, let me know!

**Comment by dcfix** [comment]
Thanks - got hung up trying to figure out git rebasing. Maybe next time.

**Pull request closed without merging**